File: FPL/salary.c
#include <stdio.h>

void main()
{
    int salary, hra, da, total_salary;

    printf("Enter your basic slary: ");
    scanf("%d", &salary);

    hra = 0.2 * salary;
    da = 1.5 * salary;
    total_salary = salary + hra + da;

    printf("Your H.R.A is: %d\n", hra);
    printf("Your D.A is: %d\n", da);
    printf("Your total salary is: %d\n", total_salary);
}

File: PPS/salary.py
basic = int(input("Enter your basic pay: "))

hra = 0.1*basic
ta = 0.05*basic

salary = basic+hra+ta

tax = 0.02*salary
net = salary-tax

print("basic salary is ", basic)
print("hra is ", hra)
print("ta is ", ta)
print("total salary is ", salary)
print("Your net salary is ", net)
